Viper Practice/src/controllers/FruitesViewController.swift

```diff
@@ -1,17 +1,16 @@
 //
-//  TasksViewController.swift
+//  FruitesViewController.swift
 //  Viper Practice
 //
 //  Created by Huzaifa Umer on 20/12/2023.
 //
 
 import UIKit
 
-class TasksViewController: UIViewController {
+class FruitesViewController: UIViewController {
```

Comment:
make the class final

Reply:
class access modifier is changed to final